pkg/sloop/webserver/webfiles/index.html

```diff
@@ -36,9 +36,10 @@
             <select name="query" id="filterquery" hidden="true"> </select>
             <!-- <br><br> -->
 
-            <label for="selectedEndTime">End time</label><br/>
+            <label for="selectedEndTime">End Time</label><br/>
             <div>
                 <input type="datetime-local" step="1" id="selectedEndTime" name="selectedEndTime" />
+                <input type="button" value="now" name="now" id="now">
```

Comment:
nit: Can you please change label to "Now"

Reply:
done